src/OpenLoco/Objects/TrainStationObject.h

```diff
@@ -31,25 +39,42 @@ namespace OpenLoco
         uint8_t flags; // 0x0C
         uint8_t var_0D;
         uint32_t image; // 0x0E
-        uint8_t pad_12[0x22 - 0x12];
+        uint32_t var_12[4];
         uint8_t num_compatible; // 0x22
         uint8_t mods[7];
-        uint16_t designed_year; // 0x2A
-        uint16_t obsolete_year; // 0x2C
-        uint8_t var_2E[16];
-        uint8_t var_3E[16];
-        uint8_t var_4E[16];
-        uint8_t var_5E[16];
+        uint16_t designed_year;            // 0x2A
+        uint16_t obsolete_year;            // 0x2C
+        std::byte* cargoOffsetBytes[4][4]; // 0x2E
         uint8_t pad_6E[0xAC - 0x6E];
 
         void drawPreviewImage(Gfx::Context& context, const int16_t x, const int16_t y) const;
         void drawDescription(Gfx::Context& context, const int16_t x, const int16_t y, [[maybe_unused]] const int16_t width) const;
+        std::vector<CargoOffset> getCargoOffsets(const uint8_t rotation, const uint8_t nibble) const;
     };
 #pragma pack(pop)
 
     namespace TrainStation::ImageIds
     {
         constexpr uint32_t preview_image = 0;
         constexpr uint32_t preview_image_windows = 1;
+        // These are relative to to var_12
+        // var_12 is the imageIds per sequenceIndex (for start/middle/end of the platform)
+        constexpr uint32_t style0StraightBackNE = 0;
```

Comment:
I'm thinking of making this into a sub namespace to remove style from the name. Its not really important for trains but for road stations it will look better as all the road and tram images can be split between their two styles at namespace level.